ethereum/p2p/src/main/java/tech/pegasys/pantheon/ethereum/p2p/rlpx/RlpxAgent.java

```diff
@@ -397,11 +397,8 @@ private boolean remoteConnectionExceedsLimit() {
                 .filter(RlpxConnection::isActive)
                 .filter(RlpxConnection::initiatedRemotely)
                 .count());
-    if (remotelyInitiatedConnectionsCount == 0) {
-      return false;
-    }
     final double fractionRemoteConnections =
-        (double) remotelyInitiatedConnectionsCount + 1 / (double) maxPeers;
+        (double) (remotelyInitiatedConnectionsCount + 1) / (double) maxPeers;
```

Comment:
Will maxPeers=0 cause issues?

Reply:
good eye!  however, turns out this actually works.  this will evaluate to `Infinity` in that case and the comparison will return `true`.